### ISSUE TITLE
[DOC]: NameError: name 'linreg' is not defined

### ISSUE THREAD

**Issue by cmougan** [saattrupdan/doubt#24]
```python
>>> from sklearn.linear_model import LinearRegression
>>> from doubt import Boot
>>> from doubt.datasets import PowerPlant
>>>
>>> X, y = PowerPlant().split()
>>> clf = Boot(LinearRegression())
>>> clf = linreg.fit(X, y)
>>> clf.predict([10, 30, 1000, 50], uncertainty=0.05)
(481.9203102126274, array([473.43314309, 490.0313962 ]))
```
Outputs the following error:
```
Traceback (most recent call last):
  File "stupid.py", line 7, in <module>
    clf = linreg.fit(X, y)
NameError: name 'linreg' is not defined
```

**Comment by cmougan** [comment]
Its just a small doc error, I will make soon a PR to fix it

**Comment by saattrupdan** [comment]
Fixed in [v4.0.0](https://github.com/saattrupdan/doubt/releases/tag/v4.0.0). Thanks!
